integration/sslyze/service/index.js

```diff
@@ -11,10 +12,16 @@ const port = process.env.SVC_PORT || '8081';
 const svcTimeout = process.env.SSLYZE_SVC_TIMEOUT || '120000'; // 2 mins
 
 router.get('/sslyze/run/:host', async (ctx) => {
+  const hostName = ctx.params.host;
+  const fileName = `${hostName}.json`;
   try {
-    const result = await exec(`cd /usr/local/bin && ./sslyze --regular --http_headers ${ctx.params.host}`);
+    const result = await exec(`sslyze --regular --http_headers ${hostName} --json_out=${fileName}`);
```

Comment:
what do you think about using --json_out=- send json output on the standard output in this way:
const result = await exec(`sslyze --regular --http_headers ${hostName} --json_out=-`);
ctx.headers = { 'Content-Type': 'application/json' };
ctx.body = JSON.parse(result.stdout);

Reply:
I thought there was some value in capturing both the formatted text output and the json output separately.
That way each of the scenario does not have to be to be rewritten immediately with the change. 
e.g. verifying heartbleed requires additional code now to get it from the json output.